Authentication/Login/View/LoginConfiguration.swift

```diff
@@ -13,7 +13,7 @@ import Foundation
 */
 public protocol LoginControllerTransitionDelegate {
```

Comment:
Shouldn't we notify `userDidLogin` here?

```swift
func didLogin(controller: LoginController, user: User)
```
or we are handling this logic in other place?